src/helpers/executionResultHelper.ts

```diff
@@ -86,6 +87,43 @@ export function findAttributeInDimension(
     );
 }
 
+export interface IGeoAttributesInDimension {
+    locationAttribute: IUnwrappedAttributeHeaderWithItems;
+    segmentByAttribute: IUnwrappedAttributeHeaderWithItems;
+}
+
+export function findGeoAttributesInDimension(
```

Comment:
This util is quite specific to only Geo chart, so should we move it under geo chart util?
like `.../geoChart/executionResultHelper.ts`

Reply:
👍

Reply:
if moved somewhere move also its UTs

Reply:
It is moved to `geoChart/executionResultHelper.ts`